lib/spacesuit/session_service.ex

```diff
@@ -139,7 +105,8 @@ defmodule Spacesuit.SessionService do
         {:ok, %HTTPoison.Response{status_code: 200, body: body}} ->
           {:ok, body}
 
-        {:ok, %HTTPoison.Response{status_code: code, body: body}} when code >= 400 and code <= 499 ->
+        {:ok, %HTTPoison.Response{status_code: code, body: body}}
+        when code >= 400 and code <= 499 ->
```

Comment:
sorry. This is from the 1.6.4 formatter. I can remove it if you want, but I recommend updating and starting to use the formatter.